cmd/kubeadm/app/cmd/token_cmd_test.go

```diff
@@ -0,0 +1,67 @@
+// +build cmd
```

Comment:
Good point. That does mean I won't be able to use the +build directive to separate the tests, but I think I can get by with just using a specific naming scheme for the tests and doing "go test -run Cmd" instead of "go test -tags cmd". I should have a new commit soon.